io.github.ec_.Quake3e.OpenArena.appdata.xml

```diff
@@ -25,7 +25,8 @@
       Domination.
     </p>
     <p>
-      This version of OpenArena is based on the Quake3e engine.
+      This version of OpenArena is based on the Quake3e engine by "Ec" on GitHub, I do not take credit for his work or any of the work done by the OpenArena team.
+      This FlatPak was made to be a replacement of the original one released on 2012, as it contains bugs and annoyances not present in this engine. The gameplay is 100% compatible and if not, please make an issue in my repo(xplshn/openarena), I'll investigate it.
```

Comment:
But if I don't, people will see OpenARENA, and OpenARENA(Quake3e) and will think "Is there really any difference" And they won't find out because they both use the same engine, one is enhanced and has bugfixes.